apollo-runtime/src/main/java/com/apollographql/apollo/ApolloCall.java

```diff
@@ -128,4 +129,26 @@ public void onCanceledError(@Nonnull ApolloCanceledException e) {
       onFailure(e);
     }
   }
+
+  /**
+   * Represents a status event that corresponds to action that happen to {@link ApolloCall}
+   */
+  enum StatusEvent {
+    /**
+     * {@link ApolloCall} scheduled for execution
+     */
+    SCHEDULED,
+    /**
+     * {@link ApolloCall} fetches response from cache
+     */
+    FETCH_CACHE,
```

Comment:
Maybe some better documentation if this occurs before the cache fetch begins, or after it completes?